src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,6 +5,9 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty lines before the class declaration.